arbnode/node.go

```diff
@@ -63,14 +62,10 @@ import (
 )
 
 type DAConfig struct {
-	Mode             string                `koanf:"mode"`
-	ReferenceDA      referenceda.Config    `koanf:"referenceda"`
 	ExternalProvider daclient.ClientConfig `koanf:"external-provider" reload:"hot"`
```

Comment:
I think we should finally take `Enable` out from `daclient.ClientConfig` here (this belongs to this level of configuration, or, even better, the one up), not to the client itself